# BasicComms.msg
# Initial format for a comms message

# Time the message was sent
time sent

# ID of the sender
int32 sender_id

# Parameter 1
float32 parameter_1

# Parameter 2
float32 parameter_2

# Parameter 3
float32 parameter_3

# Parameter 4
float32 parameter_4

# Parameter 5
float32 parameter_5

# Array of data
float32[] data_array